docs/docs/recipes/protect-your-api/README.md

```diff
@@ -2,4 +2,269 @@
+app.mount('#app');
+```
+
+</TabItem>
+<TabItem value="js" label="VanillaJs">
+
+```js
+// TODO:
+```
+
+</TabItem>
+</Tabs>
+
+Let's take a breath here. Your application is almost ready 😊. Logto SDK is well configured at this point. Users could sign in / sign up through the UI now.
+
+After end-users successfully log in, they may try to get access to some of your backend API resources through the client application upon their request. We'll need to loop Logto in again at this point. Call Logto server through our SDK to request an `access_token` before making the actual API request.
+
+:::note
+
+The requested target resource should be:
+
+1. Registered in Logto server.
+2. Initiated as configs into the Logto SDK at your application.
+
+:::
+
+<Tabs>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+logtoClient.getAccessToken("<your-target-api-resource>", { logtoException: LogtoException?, result: AccessToken? ->
+  // custom logic
+})
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+logtoClient.getAccessToken("<your-target-api-resource>", (logtoException, accessToken) -> {
+    // custom logic
+});
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+```swift
+  let token = try await client.getAccessToken(for: "<your-target-api-resource>")
+  // custom logic
+```
+
+</TabItem>
+<TabItem value="react" label="React">
+
+```js
+const { getAccessToken } = useLogto();
+const token = await getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+<TabItem value="vue" label="Vue">
+
+```js
+const { getAccessToken } = useLogto();
+const token = await getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+<TabItem value="js" label="VanillaJs">
+
+```js
+const token = await logtoClient.getAccessToken('<your-target-api-resource>');
+// custom logic
+```
+
+</TabItem>
+
+</Tabs>
+
+If the client is well-authorized, a JWT format `access_token` will be granted and issued by Logto, specifically for the requested entity. Encrypted, audience_restricted, and with lift-time. Carrying all the necessary info that can represent the authority of this request.
+
+Append the token to your request's Authorization header as the Bearer code:
+
+i.e.
+
+```bash
+GET https://logto.dev/api/users
+
+Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwiYXVkIjoiaHR0cHM6Ly9sb2d0by5kZXYvYXBpL3VzZXJzIiwiaXNzIjoiaHR0cHM6Ly9sb2d0by5kZXYvb2lkYyIsIm5hbWUiOiJKb2huIERvZSIsImlhdCI6MTUxNjIzOTAyMiwiZXhwIjoxNTE2MzI1NDIyfQ.PjIJl00YNC84EPNYLEdpEEAdAxqhekCYhFEckvRokek
+
+```
+
+Now you have your application well equipped and ready to launch 🚀 .
+
+## 3. Validate the Authorization header on your API side
+
+//TODO:
+
+We follow the standard Code Based OAuth 2.0 Authorization Protocol. If you are interested in the strategy behind Logto, please refer to OAuth 2.0's [official document](https://datatracker.ietf.org/doc/html/rfc6749#section-1.3.1) for more details.
```

Comment:
nit: just personal preference 

I commonly see `See link.`, `See link for more.`, `See link for details.` or `See link for more details.` in the tech book.
```suggestion
We follow the standard Code Based OAuth 2.0 Authorization Protocol. If you are interested in the strategy behind Logto, see OAuth 2.0's [official document](https://datatracker.ietf.org/doc/html/rfc6749#section-1.3.1) for more details.
```

Reply:
`Please refer to` looks fine to me, tho

Reply:
depends on the sentence context.  would like to keep `refer to` here.